README.md

```diff
@@ -119,7 +119,6 @@ Bind calls support the following optional [arbitrary parameters](https://docs.cl
 
 | Option | Type   | Description
 |:-------|:------ |:-----------
-| dbname | String | The name of the Database to bind the application to (it must be provisioned previously)
```

Comment:
I've remembered why I left this (and the struct below) in place. There was a thought about adding an `access_level` parameter here to allow for different access levels in future (ie read/write with no schema access, and read-only)

I need to have a conversation with @bleach and/or @Rory80Hz about this.